openlibrary/templates/books/custom_carousel.html

```diff
@@ -43,10 +46,19 @@
   <div class="book carousel__item">
     <div class="book-cover">
       <a href="$(url)">
-        <img class="bookcover"
-          width="130" height="200"
-          title="$('%s%s'%(title,byline))"
-          $img_attr="$(cover_url)"/>
+        $if cover_url:
+          <img class="bookcover"
+            width="130" height="200"
+            title="$('%s%s'%(title,byline))"
+            $img_attr="$(cover_url)"/>
+        $else:
+          <div class="carousel__item__blankcover">
+              <div>
```

Comment:
Ah, this is for the inner border; again a class would make this clearer.

Reply:
Can I ask to punt this since we'll probably need to refactor the other areas we're applying this same logic? Same with the `:before`?